### DISCUSSION TITLE
Problems using GMTSAR to run MintPy

### DISCUSSION THREAD

**Discussion by anaferreira97** [insarlab/MintPy#759] [Q&A] [answered]
I'm trying to run Mintpy using GMTSAR processing, however I can't create the .txt file that needs to be in mintpy folder, because there is no available structure example. Can you tell me where I can find an example?

**Comment by yunjunz** [comment, marked as answer]
I believe here (https://mintpy.readthedocs.io/en/latest/dir_structure/#gmtsar) is what you are looking for.

**Reply by anaferreira97** [reply]
Yes, it is. Thank you!
